packages/ERTP/src/typeGuards.js

```diff
@@ -1,23 +1,64 @@
 import { isNat } from '@agoric/nat';
+import { passStyleOf, REMOTE_STYLE } from '@agoric/marshal';
+
+const { isFrozen } = Object;
+
+// A type guard predicate named `looksLikeFoo` tests that something seems to be
+// a Foo, produces static type info on the truthy path alleging that it is a
+// Foo, but does not validate that it is a well formed Foo. Names like `isFoo`
+// should be reserved for predicates that actually validate objects coming from
+// untrusted callers.
+//
+// The corresponding assertions would be `assertLooksLikeFoo` and `assertFoo`.
+// These produce the same static type info, but on the success path rather than
+// the truthy path.
 
 /**
- * Type guard for SetValue
+ * Non-validating type guard for SetValue
+ *
  * Used as a pre-validation check to select which validator
  * (mathHelpers) to use, and also used with assert to satisfy
  * Typescript checking
  *
  * @param {Value} value
  * @returns {value is SetValue}
  */
-export const isSetValue = value => Array.isArray(value);
+export const looksLikeSetValue = value => Array.isArray(value);
 
 /**
- * Type guard for NatValue.
+ * Non-validating type guard for NatValue.
+ *
  * Used as a pre-validation check to select which validator
  * (mathHelpers) to use, and also used with assert to satisfy
  * Typescript checking
  *
  * @param {Value} value
  * @returns {value is NatValue}
  */
-export const isNatValue = value => isNat(value);
+export const looksLikeNatValue = value => isNat(value);
+
+/**
+ * Call this for a validated answer (that in this case happens to be the same).
+ *
+ * @param {Value} value
+ * @returns {value is NatValue}
+ */
+export const isNatValue = looksLikeNatValue;
```

Comment:
Should this be in the type guard file? I think we should save this file for the non-validating versions

Reply:
Six files outside ERTP import this. I was going to rename their use of this until I realized that
   * they seem to be using it to validate
   * Unlike `isSetValue` or `isValue`, `isNatValue` actually does validate.

This validating form is still a type guard, in that it produces static type info on its truthy path. So I'm inclined to leave it where it is. I don't see a more natural place for it.